fleet_management_api/__main__.py

```
@@ -1,11 +1,13 @@
-import requests
+import requests  # type: ignore
+
 import fleet_management_api.script_args as _args
 import fleet_management_api.app as app
 from fleet_management_api.api_impl.auth_controller import init_security
 from fleet_management_api.controllers.security_controller import set_auth_params
 from fleet_management_api.database.db_access import set_content_timeout_ms
 from fleet_management_api.database.connection import set_up_database
 from fleet_management_api.api_impl.data_setup import set_up_data
+from fleet_management_api.logs import configure_logging, LOGGER_NAME
```

Ruff

`fleet_management_api.logs.LOGGER_NAME` imported but unused

Remove unused import: `fleet_management_api.logs.LOGGER_NAME`

(F401)